.github/workflows/nightly.yml

```diff
@@ -27,7 +27,9 @@ jobs:
 
       - run: cargo install --force --path risc0/cargo-risczero
```

Comment:
If we install the cargo-risczero tool here do we still need to run `cargo run` below? 

Using the installed version would prevent a rebuild, and we need the cargo-risczero tool installed for the crate validator.

Reply:
good point. I reverted to this command